kvbc/include/categorization/kv_blockchain.h

```diff
@@ -220,6 +220,8 @@ class KeyValueBlockchain {
 
   // currently we are operating with single thread
   util::ThreadPool thread_pool_{1};
+  // For concurrent deletion of the categories inside a block.
+  util::ThreadPool prunning_thread_pool_{2};
```

Comment:
wouldn't it be better to make it configurable?

Reply:
I thought of making it configurable, but because we currently have only two categories that require running in their thread I decided to hard code it. But maybe it does worth the extra effort.